packages/sdk/src/types/requests/ConnectorRequest.ts

```diff
@@ -18,18 +18,84 @@ export enum ConnectorRequestStatus {
     Completed = "Completed"
 }
 
+export interface ConnectorRequestContent {
+    "@type": string;
+    "@context"?: string;
```

Comment:
Lass @context besser mal weg. Das ist ja nie gefüllt. Eigentlich sollten wir das sogar aus dem ContentJSON in der Content Library entfernen. Das verwirrt bloß.